base/libgit2/types.jl

```diff
@@ -357,28 +357,60 @@ immutable DiffDelta
 end
 DiffDelta() = DiffDelta(Cint(0), UInt32(0), UInt16(0), UInt16(0), DiffFile(), DiffFile())
 
-immutable MergeOptions
-    version::Cuint
-    flags::Cint
-    rename_threshold::Cuint
-    target_limit::Cuint
-    metric::Ptr{Void}
-    file_favor::Cint
-    file_flags::Cuint
-end
-MergeOptions(; flags::Cint = Cint(0),
-               rename_threshold::Cuint = Cuint(50),
-               target_limit::Cuint = Cuint(200),
-               metric::Ptr{Void} = C_NULL,
-               file_favor::Cint = Cint(Consts.MERGE_FILE_FAVOR_NORMAL),
-               file_flags::Cuint = Cuint(Consts.MERGE_FILE_DEFAULT)) =
-    MergeOptions(one(Cuint),
-                 flags,
-                 rename_threshold,
-                 target_limit,
-                 metric,
-                 file_favor,
-                 file_flags)
+if LibGit2.version() >= v"0.24.0"
+    immutable MergeOptions
+        version::Cuint
+        flags::Cint
+        rename_threshold::Cuint
+        target_limit::Cuint
+        metric::Ptr{Void}
+        recursion_limit::Cuint
+        # default_driver::Cstring  # TODO: added after v"0.24.1"
```

Comment:
presumably they won't backport a new field in a point release? so 0.25.0 will be the first to have this?